db/sql/00_msar.sql

```diff
@@ -4775,10 +4775,14 @@ END;
 $$ LANGUAGE plpgsql;
 
 
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anyelement);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, anyelement, boolean);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, text, boolean);
+DROP FUNCTION IF EXISTS msar.get_record_from_table(oid, numeric, boolean);
 CREATE OR REPLACE FUNCTION
 msar.get_record_from_table(
   tab_id oid,
-  rec_id anyelement,
```

Comment:
@seancolsen I did some more digging into this issue, and changing the datatype here from `anyelement` -> `anycompatible` works flawlessly.

Reply:
Oh, interesting!

Reply:
I pushed d4fba8348e5470f4633787a417f33dded28270bc to make this change. Thanks for pointing that out!